Homework6/Homework6.sln

```diff
@@ -3,7 +3,19 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.32002.261
 MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Task1", "Task1\Task1.csproj", "{FBA28ED7-91AF-47D9-8D79-2B049F943A60}"
```

Comment:
> Place the generated types in a separate class library. In a console application, connect this class
> library and test the work of types from it.

The solution should have 2 projects - attributes in library and console app that references lib.